app/models/towing_task.rb

```diff
@@ -0,0 +1,26 @@
+class TowingTask < LaunchJobTask
+  after_update :post_retry_job_task, :if => proc { state == 'completed' }
+
+  def post_retry_job_task
+    original_task = LaunchJobTask.find_by(:child_task_id => id)
```

Comment:
@hsong-rh I think we should be storing the LaunchJobTask.id in the child_task_id of the TowingTask. So that way we can figure out if we have run multiple TowingTask's for this LaunchJob and have a limit on how many 🚚 we have sent to fetch this task.